Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -613,7 +613,7 @@ section
 
 omit [IsDomain R]
 
-theorem monic_prod_multiset_X_sub_C (s : Multiset R) : Monic (s.map fun a => X - C a).prod :=
+theorem monic_multisetProd_X_sub_C (s : Multiset R) : Monic (s.map fun a => X - C a).prod :=
```

Comment:
Can you add deprecations please?
[Polynomial.monic_multiset_prod_of_monic](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Polynomial/Monic.html#Polynomial.monic_multiset_prod_of_monic) and [Polynomial.Monic.nextCoeff_multiset_prod](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Polynomial/Monic.html#Polynomial.Monic.nextCoeff_multiset_prod) should probably also be renamed, maybe in another PR.